### PR TITLE
Remote Settings: Toolbar icon & clean-up

### PR DESCRIPTION
**Pull request by mbtools**
Continue #4718

- Add "Edit" icon behind URL to jump directly to "Remote Settings"

image

- Minor fix to remember original URL
- Branch > Switch to PR: removed
- Advanced > Checkout commit: removed
- Advanced > Make Offline / Online: removed menu items and popups
- Advanced > Change Remote: removed menu item and page

### CODE REVIEW

**Comment by mbtools** [comment]
Net -714. Enough deleting for this one. 

Closes #4718

File: src/ui/zcl_abapgit_gui_chunk_lib.clas.abap [hunk 1]
```diff
@@ -812,6 +813,15 @@ CLASS zcl_abapgit_gui_chunk_lib IMPLEMENTATION.
...
 
+      IF iv_show_edit = abap_true.
+        ri_html->add_a( iv_txt   = ri_html->icon( iv_name  = 'edit-solid'
+                                                  iv_class = 'pad-sides'
+                                                  iv_hint  = 'Change remote' )
+                        iv_act   = |{ zif_abapgit_definitions=>c_action-repo_remote_settings }?| &&
+                                   |key={ io_repo->get_key( ) }|
+                        iv_class = |url| ).
+      ENDIF.
+
```
Comment (g-back) [inline src/ui/zcl_abapgit_gui_chunk_lib.clas.abap:824]: Looks like the changes from #4723 are included in the first commit?

Comment (mbtools) [reply src/ui/zcl_abapgit_gui_chunk_lib.clas.abap:824]: Repo_view is in both. I didn't want a conflict. But this one doesn't  have the css and icon mime. I can combine the PRs is you like

Comment (g-back) [reply src/ui/zcl_abapgit_gui_chunk_lib.clas.abap:824]: It'd be easier if you combined I think in this case (in a separate commit) easier to review as a whole then.
Will test this sometime today

Comment (mbtools) [reply src/ui/zcl_abapgit_gui_chunk_lib.clas.abap:824]: Done. If you test without #4721, then there will be an empty spot next to the URL but you should be able to click it. Or simply pull both #4721 and #4724.

**Comment by g-back** [comment]
Looks good overall: syntax check shows no errors, abaplint shows no issues, no dumps or errors when using the functionalities.
Unless anyone knows any more scenarios that should be tested?

One thing: I'd add the edit Button also for offline repos, next to the name. This can be a follow-up issue/PR though.

**Comment by mbtools** [comment]
Thanks for testing. "edit" icon should now work for offline, too.

**Review by g-back — APPROVED** [review]
👍 (Merge after #4721)

**Comment by larshp** [comment]
-700 lines 💪

**Comment by sbcgua** [comment]
Cool :+1: Much cleaner.

I would just suggest recoloring the edit icon, it loos like disabled
image
image

**Pull request merged**